tests/components/pooldose/conftest.py

```diff
@@ -75,6 +75,19 @@ def mock_config_entry(device_info: dict[str, Any]) -> MockConfigEntry:
     )
 
 
+@pytest.fixture
+def mock_migration_config_entry(device_info: dict[str, Any]) -> MockConfigEntry:
```

Comment:
```suggestion
def mock_config_entry_v1_1(device_info: dict[str, Any]) -> MockConfigEntry:
```
nit: make it clearer from the name, so that reading the docstring becomes unecessary. Also future proof if later we need more mocks for other migrations.